cli/pkg/local/server.go

```diff
@@ -309,6 +309,15 @@ func (s *Server) DeployProject(ctx context.Context, r *connect.Request[localv1.D
 		}
 	}
 
+	if s.app.ch.Org != r.Msg.Org {
+		// Switching to passed org
+		s.app.ch.Org = r.Msg.Org
```

Comment:
Should this be a method on `cmdutil.Helper`? That way we can update any state around org in a single place.